CHANGES.md

```diff
@@ -1506,9 +1506,9 @@
   [Also part of PR 431](https://github.com/nextstrain/augur/pull/431)
 * traits: Allow input of `--weights` which references a `.tsv` file in the following format:
   ```
-  division	Hubei	10.0
-  division	Jiangxi	1.0
-  division	Chongqing	1.0
+  division  Hubei   10.0
```

Comment:
I think these tab-to-spaces conversions are automatic from your editor, right? They shouldn't be part of this commit (also some other unintended whitespace changes on lines 1345-1347)

Reply:
Ah geez, didn't see those, sorry. Thanks for catching.